### PR TITLE
Ruc/demandtech/button with multiple type

### LINKED ISSUE #68: Add Button with multiple type
As per [Figma](https://www.figma.com/file/4hLGdFmyqJqwO64s6xi6m8/Crypto-Components-(Community)?node-id=115%3A604&t=WD2flZRh8w4EVeeg-4)

# Preview

<div align="center">

image

</div>

### PR DESCRIPTION
**Pull request by Demandtech**
fixes #68 
screencapture-localhost-3000-stories-buttons-2022-12-15-14_42_47
screencapture-localhost-3000-stories-buttons-2022-12-15-14_44_40

### CODE REVIEW

File: src/routes/buttons/buttons.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M10.0002 1.66671C8.35198 1.66671 6.74082 2.15545 5.37041 3.07113C4 3.9868 2.9319 5.28829 2.30117 6.81101C1.67044 8.33373 1.50541 10.0093 1.82695 11.6258C2.14849 13.2423 2.94217 14.7272 4.1076 15.8926C5.27304 17.058 6.7579 17.8517 8.37441 18.1733C9.99092 18.4948 11.6665 18.3298 13.1892 17.699C14.7119 17.0683 16.0134 16.0002 16.9291 14.6298C17.8448 13.2594 18.3335 11.6482 18.3335 10"
+            strokeWidth="3"
+            strokeLinecap="round"
+            strokeLinejoin="round"
+          />
+        </svg>
```
Comment (ranaintizar) [inline src/routes/buttons/buttons.jsx:80]: What do you think about extracting svg in a file?
Like example.svg

**Review by ranaintizar — APPROVED** [review]
Excellent PR, @Demandtech :muscle: :100:

File: src/routes/buttons/buttons.jsx (hunk 1 above)
Comment (Demandtech) [reply src/routes/buttons/buttons.jsx:80]: @ranaintizar but the SVGs are not the same color on all button, is there any way I can change the colors if I extract it?

Comment (ranaintizar) [reply src/routes/buttons/buttons.jsx:80]: @Demandtech You can add a Class on svg by importing as a component.

Comment (Demandtech) [reply src/routes/buttons/buttons.jsx:80]: Oh work thank you, I will check it out, I will update you once I'm done with it;

Comment (Demandtech) [reply src/routes/buttons/buttons.jsx]: @ranaintizar Done! Please review again

**Comment by Demandtech** [comment]
@mrxshinji @aftabrehan all corrections made

File: src/routes/buttons/buttons.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import './buttons.css'
+import { ReactComponent as Loading } from '../../assets/svg/btn-loading.svg'
```
Comment (ranaintizar) [inline src/routes/buttons/buttons.jsx:2]:
```suggestion
import { ReactComponent as Loading } from 'assets/svg/btn-loading.svg'
```

File: src/index.js [hunk 3]
```diff
@@ -10,19 +10,19 @@ import StoryPage from './routes/story-page/story-page'
 import Homepage from './routes/homepage/homepage'
 import TestPage from './routes/test-page.jsx/test-page'
 import ComponentStory from './routes/story-page/componet'
```
Comment (ranaintizar) [inline src/index.js:12]:
```suggestion
import Homepage from 'routes/homepage/homepage'
import TestPage from 'routes/test-page.jsx/test-page'
import ComponentStory from 'routes/story-page/componet'
```

File: src/index.js [hunk 4]
```diff
@@ -10,19 +10,19 @@ import StoryPage from './routes/story-page/story-page'
...
-import ButtonStory from './routes/story-page/button'
 import QuickTransfer from './components/quickTransfer/QuickTransfer'
 import AnotherProfile from './components/anotherProfile/anotherProfile'
 import NotificationStory from './routes/notification-page/index'
 import Payment from './routes/payment/payment'
 import Cookie from './routes/cookie/cookie'
 import SidebarMenuComponent from './routes/sidebarmenu-component/sidebarmenu'
 import CardPayment from './routes/card-payment/CardPayment'
```
Comment (ranaintizar) [inline src/index.js:19]:
```suggestion
import QuickTransfer from 'components/quickTransfer/QuickTransfer'
import AnotherProfile from 'components/anotherProfile/anotherProfile'
import NotificationStory from 'routes/notification-page/index'
import Payment from 'routes/payment/payment'
import Cookie from 'routes/cookie/cookie'
import SidebarMenuComponent from 'routes/sidebarmenu-component/sidebarmenu'
import CardPayment from 'routes/card-payment/CardPayment'
```

File: src/index.js [hunk 5]
```diff
@@ -10,19 +10,19 @@ import StoryPage from './routes/story-page/story-page'
...
-import CodeWriter from './routes/mock-code-writer/code-writer'
 import Profile from 'routes/profile/profile'
+import CoinValue from 'routes/coin-value/coinValue'
+import CodeWriter from './routes/mock-code-writer/code-writer'
```
Comment (ranaintizar) [inline src/index.js:22]:
```suggestion
import CodeWriter from 'routes/mock-code-writer/code-writer'
```

**Review by ranaintizar — APPROVED** [review]
Excellent PR, @Demandtech :muscle: :100: 

Just needs minor fixes and then it's good to go.

File: src/routes/buttons/buttons.jsx (hunk 2 above)
Comment (Demandtech) [reply src/routes/buttons/buttons.jsx:2]: @ranaintizar @mrxshinji @aftabrehan   I got an error when I made this change likewise the rest of the  suggested changes

screencapture-localhost-3000-stories-buttons-2022-12-17-18_41_40

Comment (ranaintizar) [reply src/routes/buttons/buttons.jsx:2]: @Demandtech Remove the first slash"/" before assets.

**Comment by Demandtech** [comment]
@ranaintizar I think I'm done now

File: src/index.js [hunk 6]
```diff
@@ -7,23 +7,23 @@ import App from './App'
 
 import ErrorPage from './routes/error-page/error-page'
 import StoryPage from './routes/story-page/story-page'
```
Comment (ranaintizar) [inline src/index.js:9]:
```suggestion
import ErrorPage from 'routes/error-page/error-page'
import StoryPage from 'routes/story-page/story-page'
```

**Review by ranaintizar — APPROVED** [review]
Excellent PR, @Demandtech :muscle: :100:

**Review by aftabrehan — APPROVED** [review]
Excellent PR, @Demandtech 💪 🚀 💯

**Pull request merged**